rs/execution_environment/Contributing.md

```diff
@@ -6,6 +6,9 @@
 4. Motoko needs to be updated: [Motoko](https://github.com/dfinity/motoko). Inform *@eng-motoko* of the work required.
 5. Add any new types or update existing ones in `types/management_canister_types`.
 6. Implement the new API in `execution_environment` crate. Main parts that need to be updated: the core implementation in the `canister_manager` module as well as wiring up the new API in `ExecutionEnvironment` so that it is accessible from the outside.
-7. Tests: add any required tests to cover the new or updated functionality. Typically, these tests can be written using the `ExecutionTest` framework. In some cases, you might need to use State Machine tests, e.g. if you would need to test things like inter-canister calls or special features like HTTPS outcalls or threshold signatures that require interaction with the Consensus layer.
-8. Once the *Interface Spec* PR has been agreed on, [Motoko](https://github.com/dfinity/motoko) and [Rust CDK](https://github.com/dfinity/cdk-rs) can be updated to use the new API on a feature branch. Coordinate with *@eng-sdk* and *@eng-motoko* as needed.
-9. Once the implementation is rolled out fully on mainnet, the Interface Spec PR, Rust CDK, and Motoko changes can be merged to master.
\ No newline at end of file
+Except for endpoints that have dedicated logic for dealing with rollback after failures (e.g., the `install_code`/`install_chunked_code` endpoints), there should always be a clear point during execution of every other endpoint so that
```

Comment:
I would like to reformulate this part as it's a bit hard to understand at first.  

 Would you agree to this? 
 ```suggestion
Note: Some management methods, such as install_code and install_chunked_code, include dedicated logic to handle rollback after failures. However, for the majority of methods, there should be a clearly defined point during execution at which changes become permanent and rollback is no longer possible. Specifically:
```